server/auth/github/github.go

```diff
@@ -22,12 +22,17 @@ const (
 type Config struct {
 	config       *oauth2.Config
 	organization string
+	whitelist    map[string]bool
 }
 
 // New creates a new Github provider from a configuration.
 func New(c *config.Auth) (auth.Provider, error) {
-	if c.ProviderOpts["organization"] == "" {
-		return nil, errors.New("github_opts organization must not be empty")
+	var users_whitelist = map[string]bool{}
```

Comment:
use `usersWhitelist := make(map[string]bool)`

I need to add the linter to the test runner 😢